portfolio/src/main/webapp/blogScript.js

```diff
@@ -5,11 +5,33 @@ window.addEventListener('DOMContentLoaded', (event) => {
 function getComments() {
     fetch('/data').then(response => response.json()).then((comments) => {
         
-        const commentElement = document.getElementById('comment-container');
-        commentElement.innerHTML = '';
-
-        for (let comment of comments) {
-            commentElement.append(comment +  ", ");           
-        }
+        const commentListElement = document.getElementById('comment-list');
+        comments.forEach((comment) => {
```

Comment:
A for-of loop is almost always preferred over a forEach loop because it's not inverted and is easier to debug since it doesn't spawn callbacks that require separate callbacks.